charts/eks-anywhere-packages/README.md

```diff
@@ -0,0 +1,65 @@
+# eks-anywhere-packages
+
+A Helm chart for eks-anywhere-packages, a controller for managing the lifecycle of EKS Anywhere packages.
+
+![Version: 0.0.0](https://img.shields.io/badge/Version-0.0.0-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: 0.0.0](https://img.shields.io/badge/AppVersion-0.0.0-informational?style=flat-square)
+
+## Installing the Chart
+
+Before the chart can be installed the we need to install cert-manager.
+
+```bash
+kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.7.1/cert-manager.yaml
+```
+
+To do a simple install run:
+
+```bash
+helm upgrade --install --namespace eksa-packages --create-namespace \
+  eksa-packages eks-anywhere-packages/ \
+  --values values.yaml
+```
+
+To Uninstall:
+
+```
+helm delete eksa-packages
+```
+
+## Values
+
+| Key | Type | Default | Description |
```

Comment:
Is these values section required in the README? It clutters the README without providing usability of the application in question.

Reply:
It's pretty standard practice for helm charts, it's from an autogenerated docs tool. Once you see it in Github it looks alot better. It will look like this https://github.com/aws/karpenter/tree/main/charts/karpenter